### DISCUSSION TITLE
Hunter Class

### DISCUSSION THREAD

**Discussion by Hpalaboy** [TrinityCore/TrinityCore#29193] [General]
hi. am i correct to say that "Revive Pet" for Hunter is not available? i have to manually put current HP in the database in order to revive my hunter pet?

**Comment by Aokromes** [comment]
you don't say witch branch, but on 3.3.5a it's available.

**Reply by Hpalaboy** [reply]
sorry my first time reporting, yes it is 3.3.5a. i attached screenshot both for the trinity core and other server hunter Beastmastery talent. as i was looking at the beastmastery talent, there is also no "Feed Pet" skill. 
![Revive Pet missing](https://github.com/TrinityCore/TrinityCore/assets/89121265/8d0f75d2-38cb-4ace-a741-61b09a7985ed)
![Revive Pet available](https://github.com/TrinityCore/TrinityCore/assets/89121265/6d78c826-777d-45e5-acec-5f6eb8861aba)

**Comment by Aokromes** [comment]
you need to make one quest to get it

**Reply by Hpalaboy** [reply]
oh i see. i thought the command ".learn my quest" will be able to get it. thanks!

**Reply by Hpalaboy** [reply]
what quest do i need to do? the hunter class chain quest?

Edit: got the revive pet/feed pet skills after completing the hunter class chain quest.
